### PR TITLE
Feature/scale manager test page

### PR DESCRIPTION
**Pull request by 902seanryan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,15 +11,15 @@
     "bellhop-iframe": "^2.1.2",
     "lodash-es": "^4.17.10",
     "quill": "^1.3.6",
+    "springroll": "git+https://github.com/SpringRoll/SpringRoll.git#v2",
```
Comment (chipbell4) [inline package.json]: Let's point at a different branch. I want to blow away that v2 branch.

File: src/views/ScaleManagerDemo.vue [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+      <a class="scaleManager__source"  href="https://github.com/SpringRoll/springroll-io-safe-scale-manager-demo/tree/feature/safe-scale-manager-demo">Source Code for the demo game</a>
+    </div>
+    <div class="scaleManager__events">
+      <h2 class="scaleManager__header">Scaling Options</h2>
+      <v-form
+      ref="scaleForm"
+      v-model="scaleValid"
+      >
+      <template v-for="value in scaleValues">
+        <!-- <label class="scaleManager__label" :for="value.label" :key="value.label">{{ value.label }}: -->
```
Comment (chipbell4) [inline src/views/ScaleManagerDemo.vue]: Let's remove this commented out chunk.

File: src/views/ScaleManagerDemo.vue [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+        <v-text-field
```
Comment (chipbell4) [inline src/views/ScaleManagerDemo.vue:19]: I'm seeing some weird behavior in my browser when the element gains focus. The label underneath shows up which causes text to reflow in a fairly jarring way. I think there's some missing `max-width` somewhere causing the labels underneath to example this container.

File: src/views/examples/ColorFilter.vue [hunk 4]
```diff
@@ -32,6 +32,7 @@ export default {
   },
   methods: {
     changeFilter(e) {
+      console.log(e);
```
Comment (chipbell4) [inline src/views/examples/ColorFilter.vue]: Let's drop this log.

File: src/views/ScaleManagerDemo.vue [hunk 5]
```diff
@@ -0,0 +1,295 @@
...
+      });
+    },
+    validateAnchor () {
+      if (!this.$refs.anchorForm.validate()) {
+        return;
+      }
+
+      this.updateAnchor();
+    },
+    validateScale () {
```
Comment (chipbell4) [inline src/views/ScaleManagerDemo.vue:203]: Do we actually need to tie this to a button click? Can we just run this on `change` events from each of the elements?

**Review by chipbell4 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (902seanryan) [reply package.json]: Updated to just point at the default branch, assuming that's ok

File: src/views/ScaleManagerDemo.vue (hunk 5 above)
Comment (902seanryan) [reply src/views/ScaleManagerDemo.vue:203]: Updated to have the anchor update in real time. The scaling values (width/height etc) should probably be tied to the update button as the game has to re-initialize every time they change. But I'm unsure of how to convey the difference in behavior.

File: package.json [hunk 6]
```diff
@@ -11,15 +11,15 @@
     "bellhop-iframe": "^2.1.2",
     "lodash-es": "^4.17.10",
     "quill": "^1.3.6",
+    "springroll": "git+https://github.com/SpringRoll/SpringRoll.git",
```
Comment (chipbell4) [inline package.json]: Let's point at a tag for this. `2.2.0` or something

File: src/views/ScaleManagerDemo.vue [hunk 7]
```diff
@@ -0,0 +1,320 @@
...
+          label="Anchor Direction Y:"
+          type="number"
+          max="1"
+          min="-1"
+          step="0.1"
+          :rules="anchorDirectionRules"
+          hint="A normalized value for a vertical edge of the viewable area. Should be between -1 and 1."
+          @input="validateAnchor"
+        ></v-text-field>
+      <!-- <v-btn @click="validateAnchor" :disabled="!anchorValid" block color="primary" class="scaleManager__event scaleManager__button --capital font-16 font-semi-bold">Update Anchor</v-btn> -->
```
Comment (chipbell4) [inline src/views/ScaleManagerDemo.vue]: Can we delete this commented out code?

File: package.json (hunk 1 above)
Comment (902seanryan) [reply package.json]: Update to point at 2.2.0

**Review by chipbell4 — APPROVED** [review]

**Review by amalley — APPROVED** [review]

**Review by chipbell4 — APPROVED** [review]

**Review by amalley — APPROVED** [review]

**Pull request merged**